taprpc/mintrpc/mint.proto

```diff
@@ -187,6 +187,14 @@ message MintAsset {
     a BIP-86 key will be derived from the underlying wallet.
     */
     taprpc.ScriptKey script_key = 12;
+
+    /*
+    Decimal display is a single integer value that represents how fine grained
+    an asset unit is. The value of this field is the number of decimal places
+    that this asset can be divided into. For example, if the decimal_display
+    value is 2, a unit is considered to be 100 of these minted assets.
```

Comment:
Add something like "This is intended as information to wallets that display balances only and have no impact on the behavior or other parts of the protocol. The value will be added as a field to the meta data JSON of the asset (and is therefore only compatible with the JSON meta data type)."